app/models/caffeinate/mailing.rb

```diff
@@ -97,7 +97,7 @@ def user
     # Assigns attributes to the Mailing from the Drip
     def from_drip(drip)
       self.send_at = drip.send_at(self)
-      self.mailer_class = drip.options[:mailer_class]
+      self.mailer_class = drip.options[:mailer_class] || drip.options[:action_class]
```

Comment:
hate this. move to just assuming action_class

Reply:
Might be _too_ generic here, but what about just `deliverer`?

Reply:
same idea. just need to get away from mailer_class in v3